### DISCUSSION TITLE
popPK Dose simulation and PK paramters calculation

### DISCUSSION THREAD

**Discussion by AprilCCC** [metrumresearchgroup/mrgsolve#1139] [General]
Hi ,

Dose mrgsolvwcan do popPK dose simulation from observed real data, and do the PK calculation based on the simulated results? A similar result shows below:
I've looked at all the samples from the mrgsolve, but couldn't get any clue to do the simulation like this. Any idea would be appreciated!

Figure 1 observed data compared with simulated data
Figure 2 PK parameters ratio between observed and simulated data
![image](https://user-images.githubusercontent.com/87253631/160987804-4d11d150-73ea-4e6e-8b71-1c6be68ced99.png)
![image](https://user-images.githubusercontent.com/87253631/160987812-76879adc-a500-4393-b65d-0f4b9a4669ca.png)

**Comment by Mutaz94** [comment]
What you can do is to create a model, simulate different times (replicate). Create median and prediction interval. Then lay off the observed data on that plot.

**Comment by AprilCCC** [comment]
> What you can do is to create a model, simulate different times (replicate). Create median and prediction interval. Then lay off the observed data on that plot.

Thanks, I will try it again.

**Comment by liangqiongyue** [comment]
> What you can do is to create a model, simulate different times (replicate). Create median and prediction interval. Then lay off the observed data on that plot.

Hi Mutaz94，Even if the simulation is repeated, no matter how many times, as long as the dosing method is set to be the same, the concentration will be the same, so what is the point of repeating the simulation? Thank you very much, if you can help me.

**Comment by smouksassi** [comment]
in coveffectsplot vignettes I show how you can use mrgsolve simulate and do PK parameters ratios I do not cover the case of vpc as these have specialized tools but nothing preventing you from bringing in your model data and use it as a template to simulate in mrgsolve with or without uncertainties and for how many replicate you need.

**Comment by Mutaz94** [comment]
> > What you can do is to create a model, simulate different times (replicate). Create median and prediction interval. Then lay off the observed data on that plot.
> 
> Hi Mutaz94，Even if the simulation is repeated, no matter how many times, as long as the dosing method is set to be the same, the concentration will be the same, so what is the point of repeating the simulation? Thank you very much, if you can help me.

@liangqiongyue if you have variability e.g., ETAs and EPS. every time you create a replicate it should not be the same since you are drawing ETAs from multivariate distribution, thus your individual parameter should change and by that, your concentration. It will be the same if you are conducting deterministic simulations (the variances are zero).

**Comment by liangqiongyue** [comment]
> > > What you can do is to create a model, simulate different times (replicate). Create median and prediction interval. Then lay off the observed data on that plot.
> > 
> > 
> > Hi Mutaz94，Even if the simulation is repeated, no matter how many times, as long as the dosing method is set to be the same, the concentration will be the same, so what is the point of repeating the simulation? Thank you very much, if you can help me.
> 
> @liangqiongyue if you have variability e.g., ETAs and EPS. every time you create a replicate it should not be the same since you are drawing ETAs from multivariate distribution, thus your individual parameter should change and by that, your concentration. It will be the same if you are conducting deterministic simulations (the variances are zero).

Thank you for your reply. It's true that I didn't define $OMEGA and $SIGMA in the model as you said, now I know how to define them. Thank you again.
